File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var nums1=[];
    for(var i=0;i<nums.length;i++){
        for(var j=i+1;j<nums.length;j++){
            if(nums[i]+nums[j]===target){
                nums1.push(i);
                nums1.push(j);
            }
        }
    }
    return nums1;
    
};

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>ans;
        int n=nums.size();
        for(int i=0;i<n;i++){
            int k=nums[i];
            for(int j=i+1;j<n;j++){
                int sum=k+nums[j];
                if(sum==target){
                    ans.push_back(i);
                    ans.push_back(j);
                    break;
                }
            }
        }
    return ans;    
    }
};